app/components/case-studies-components/CaseStudiesOtherCaseStudies.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import Image from "next/image";
+import Link from "next/link";
+
+import { CaseStudyHeaderProps } from "@/utils/interfaces";
+
+const truncateString = (str: string, num: number): string => {
+  if (str.length <= num) {
+    return str;
+  }
+  return str.slice(0, num) + "...";
+};
+
+const CaseStudiesOtherCaseStudies: React.FC<CaseStudyHeaderProps> = ({
+  otherProjects,
+}) => {
+  return (
+    <section className="flex w-full flex-col items-center bg-white800 px-6 py-9 dark:bg-black300 md:py-[4.5rem]">
+      <div className="w-full max-w-4xl">
+        <p className="text-xs font-semibold text-primaryLight dark:text-primaryDark md:text-sm">
+          Projects
+        </p>
+        <p className="mt-2.5 text-[1.75rem] font-semibold text-black200 dark:text-white900 md:text-[2rem]">
+          Other Case Studies
+        </p>
+        <div className="mt-9 flex w-full max-w-4xl flex-col gap-6 md:mt-10 md:flex-row">
+          {otherProjects?.map((project) => (
+            <div
+              key={project.title}
+              className="flex flex-col rounded-2xl bg-white900 dark:bg-black200 md:w-1/2 md:rounded-3xl"
+            >
+              {project.figmaBannerImage && (
+                <div className="flex md:h-[16rem]">
+                  <Image
+                    src={project.figmaBannerImage}
+                    alt="Image of similar case study"
+                    className="h-full w-auto rounded-t-2xl md:rounded-t-3xl"
+                    style={{
+                      objectFit: "cover",
+                    }}
+                  />
+                </div>
+              )}
```

Comment:
I apologize if I am over-looking this but I don't see your refactored code. I still see your original code. If you haven't addressed, please be sure to refactor this section...By doing so, you'll not only streamline your code—potentially reducing it by 15 lines—but also enhance maintainability and readability. 
 
 `<ProjectCard key={project.title} project={project} />`
 
 Forgive me if I am overlooking your update.

Reply:
Sorry I think I misunderstood before. I've fixed it now. Way more resuable